nextcord/ui/view.py

```diff
@@ -505,16 +505,21 @@ def __init__(self, state: ConnectionState):
         self._synced_message_views: Dict[int, View] = {}
         self._state: ConnectionState = state
 
-    @property
-    def persistent_views(self) -> Sequence[View]:
-        # fmt: off
-        views = {
-            view.id: view
-            for (_, (view, _)) in self._views.items()
-            if view.is_persistent()
-        }
-        # fmt: on
-        return list(views.values())
+    def all_views(self) -> Sequence[View]:
+        views = [v for (v, _) in self._views.values()]
+        return views
+
+    def views(self, persistent: bool = True) -> Sequence[View]:
+        views = self.all_views()
+
+        # I'm not sure if there is a better way to do this
+        check: Callable[[View], bool]
+        if persistent:
+            check = lambda v: v.is_persistent()
+        else:
+            check = lambda v: not v.is_persistent()
+
+        return list(filter(check, views))
```

Comment:
```suggestion
        return [v for v in views if v.is_persistent() ^ not persistent] 
```
I think this should work the same, needs testing...

Reply:
I tested it, and it works the same.